datafusion/functions-aggregate/src/lib.rs

```diff
@@ -109,13 +109,13 @@ mod tests {
         let mut names = HashSet::new();
         for func in all_default_aggregate_functions() {
             assert!(
-                names.insert(func.name().to_string()),
+                names.insert(func.name().to_string().to_lowercase()),
```

Comment:
Can we not convert to lowercase here? If someone changes the name to uppercase again, the function in the function map is not the same as the function name, and it is easy to introduce a bug.

Reply:
Thanks @jayzhan211 
I think `to_lowercase` is used to ensure that the function name and its alias won't be duplicates when case-insensitive (e.g., unquoted identifier). If someone changes the function name to uppercase, the test will pass. However, they may find that they can't query this function in lowercase. They might try to add a lowercase alias or change the function name back to lowercase. I think this is the purpose of this test.

Reply:
Oh, it is the test, then it is fine